src/utils/Utils.js

```diff
@@ -287,10 +287,11 @@ function filterExamples(programlist, programType, offeringType) {
   return offeringList;
 }
 
-function errorSchemaCheck(err) {
+function errorSchemaCheck(err, communicationMode) {
   let schemaValidationResult;
```

Comment:
all the places in FCA where  errorSchemaCheck is being called has been changed right ?>

Reply:
yes, all the places are updated with 2 parameters and also tested without passing communication mode at this time it will take the SDK error schema.